src/lib/redux/siteDataSlice.ts

```diff
@@ -23,6 +23,7 @@ export const EMPTY_SITE: SiteRecord = {
   financialSummaryIds: [],
   financialSummaries: [],
   tariffPlans: [],
+  inventoryIds: [],
```

Comment:
Sorry I forgot if we needed this or not. When will this be used?

Reply:
Good catch! Making it optional + removing it here